content/modules/redisgears/installing-redisgears.md

```diff
@@ -0,0 +1,144 @@
+---
+Title: Installing RedisGears 
+description:
+weight: 60
+alwaysopen: false
+categories: ["Modules"]
+---
+Before you can use RedisGears, you have to install the RedisGears module on your RS cluster.
+
+## Minimum Requirements
+
+- Redis Enterprise 6.0.0 or later
+- The [cluster is setup]({{< relref "/rs/administering/cluster-operations/new-cluster-setup.md" >}}) and all of the nodes are joined to the cluster
+
+{{< note >}}
+For installing on Redis Enterprise Kubernetes deployments or Docker containers, you must use Ubuntu 18.04.4 LTS as the base OS because the RedisGears binaries are built using the binaries for that OS by [default](https://github.com/RedisLabs/redis-enterprise-k8s-docs#basic-installation), unless the RedisGears release specifies that there is also support for RHEL 7 docker images.
+{{< /note >}}
+
+## Installing RedisGears
+
+### Step 1: Install RedisGears dependencies
+
+On each node in the Redis Enterprise cluster:
+
+1. [Download](https://redislabs.com/download-center/modules/) the RedisGears Module - Dependencies Package from the Redis Enterprise Software section of the Downloads page.
+1. Copy the dependencies package to a node in your cluster.
+1. Run these commands as root:
+
+    ```src
+    # source /etc/opt/redislabs/redislabs_env_config.sh
+    # mkdir -p $modulesdatadir/rg/10000/deps/
+    # tar -xvf /home/ubuntu/redisgears-dependencies.linux-bionic-x64.1.0.0.tgz -C $modulesdatadir/rg/10000/deps
+    # chown -R $osuser $modulesdatadir/rg
+    ```
+
+{{< note >}}
+You must also run these commands on new nodes before you join the node to the cluster.
+{{< /note >}}
+
+### Step 2: Install the RedisGears module
+
+1. [Download](https://redislabs.com/download-center/modules/) the RedisGears Module.
+1. In the RS admin console, [add the RedisGears module]({{< relref "/modules/add-module-to-cluster.md" >}}) to the cluster.
+
+### Step 3: Create a Database and Verify the Installation
+
+1. Create a Redis Enterprise database [with the RedisGears module]({{< relref "/modules/add-module-to-database.md" >}}).
+1. To see that the database was created successfully and shards are running, run: `rladmin status`
+1. To see that RedisGears is running correctly, run: `redis-cli RG.PYEXECUTE "GearsBuilder().run()"`
```

Comment:
@bmansheim is this clear this can be done on the client side?  It does not include e.g. that it should connect to the database that you just created e.g. redis-cli -host ... -port ...